pkg/serverless/daemon/routes.go

```diff
@@ -86,6 +87,7 @@ type EndInvocation struct {
 
 func (e *EndInvocation) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	log.Debug("Hit on the serverless.EndInvocation route.")
+	e.daemon.hitOnEnd = true
```

Comment:
You're gonna want to put these behind a mutex, since each incoming request is happening in its own goroutine.

Reply:
aha, gotcha 👍